backend/project404_t8/API/api_viewsets.py

```diff
@@ -312,15 +312,15 @@ def list(self, request):
     def retrieve(self, request, pk=None):
         # permission_classes = (IsAuthenticated,)
         queryset = Post.objects.filter(pk=pk)
-        serializer_class = PostSerializer(queryset, many=True)
-
-        # response = OrderedDict()
-        # response.update({"query":"getPost"})
-        # response.update({"postID":})
-        # response.update({"url": })
-        # response.update({"previous": None})
+        # serializer_class = PostSerializer(queryset, many=True)
+        post = getPostData(request, pk=pk)
+        response = OrderedDict()
+        response.update({"query":"getPost"})
+        response.update({"post":post})
```

Comment:
I think this is supposed to be "posts" not post?

Reply:
the spec is a little ambiguous but i think "posts" is safer of a key

Reply:
I agree, should keep it consistent